pex/common.py

```diff
@@ -290,16 +290,19 @@ def link(self, src, dst, label=None):
     safe_copy(abs_src, abs_dst, overwrite=False)
     # TODO: Ensure the target and dest are the same if the file already exists.
 
-  def write(self, data, dst, label=None, mode='wb'):
+  def write(self, data, dst, label=None, mode='wb', perm=None):
     """Write data to ``chroot/dst`` with optional label.
 
     Has similar exceptional cases as ``Chroot.copy``
     """
     dst = self._normalize(dst)
     self._tag(dst, label)
     self._ensure_parent(dst)
-    with open(os.path.join(self.chroot, dst), mode) as wp:
+    _file = os.path.join(self.chroot, dst)
```

Comment:
it'd be great to avoid the use of `_` prefixed variable names here with `_file` (e.g. `file_path` would be better/more descriptive). theres no great reason nor benefit to it as far as I can tell and it's visually confusing with the private function/method convention..